sources/academy/tutorials/php/using_apify_from_php.md

```diff
@@ -0,0 +1,271 @@
+---
+title: Use Apify from PHP
+description: Learn how to access Apify's REST API endpoints from your PHP projects using the guzzle package. Follow a tutorial to run an actor and download its data.
+menuWeight: 3.93
+paths:
```

Comment:
you will need to migrate this to the new format

this is the script we used back then: https://github.com/apify/apify-docs-v2/blob/master/tools/convert.mjs

Reply:
Yeah, I've added few more changes.